src/index.ts

```diff
@@ -13,11 +13,12 @@
  */
 /* eslint-disable no-var,vars-on-top */
 // eslint-disable-next-line prefer-destructuring
-var createContext = require('react').createContext;
+import type { Context } from 'react'
+import { createContext } from 'react';
 
-var contexts = {};
+export var contexts: {[key: string]: Context<any>} = {};
```

Comment:
I believe this was previously not exported - should we keep it that way?

```suggestion
var contexts: {[key: string]: Context<any>} = {};
```